docs/stories/utf8_identifier.md

```diff
@@ -0,0 +1,8 @@
+# UTF-8 Characters within Identifiers
+
+The different Test-Editor languages (tcl, tml, config, aml) make use of identifiers that are (transitively) compiled into Java identifiers. In order to parse those accurately, the corresponding parsing rules need to be adjusted to the UTF-8 characters that should be usable. Be sure to allow only characters that satisfy `Character.isUnicodeIdentifierPart`, otherwise Java compiler errors might occur.
```

Comment:
Not sure what is meant by "transitively" here…

Reply:
transitively because e.g. tml is not compiled into java unit tests itself, but its usage within a tcl will result in generated code when the tcl itself is generated into a unit test class